### PR TITLE
Update affected version format

### PR DESCRIPTION
**Pull request by TristanCacqueray**
## hsec-tools

- [x] Previous advisories are still valid

Related: #3

### CODE REVIEW

**Comment by TristanCacqueray** [comment]
That's odd the tests passed, I was expecting a parse error.

File: .github/workflows/check-advisories.yml [hunk 1]
```diff
@@ -19,12 +19,12 @@ jobs:
...
+      - name: Run advisories checks
         run: |
           cd source
           RESULT=0
-          find advisories EXAMPLE_ADVISORY.md -type f -name "*.md" | while read FILE ; do
+          while read FILE ; do
             echo -n "$FILE: "
             hsec-tools check < "$FILE" || RESULT=1
-          done
+          done < <(find advisories EXAMPLE_ADVISORY.md -type f -name "*.md")
```
Comment (frasertweedale) [inline .github/workflows/check-advisories.yml]: I am curious why the old form did not result in exit status = 1 with bad advisory data, but the new form acts correctly.  Shell is weird.  Do you know what is the reason?

Comment (frasertweedale) [reply .github/workflows/check-advisories.yml]: Here's the answer: https://stackoverflow.com/questions/16854280/a-variable-modified-inside-a-while-loop-is-not-remembered/16854326#16854326

Quite the "gotcha".  Thanks for detecting and fixing this bug!

File: .github/workflows/check-advisories.yml [hunk 2]
```diff
@@ -23,8 +23,10 @@ jobs:
         run: |
           cd source
           RESULT=0
+          # Remove the begining of the README to extract the example.
+          (echo '```toml'; sed -e '1,/```toml/d' README.md) > EXAMPLE_README.md
           while read FILE ; do
             echo -n "$FILE: "
             hsec-tools check < "$FILE" || RESULT=1
-          done < <(find advisories EXAMPLE_ADVISORY.md -type f -name "*.md")
+          done < <(find advisories EXAMPLE_README.md EXAMPLE_ADVISORY.md -type f -name "*.md")
```
Comment (frasertweedale) [inline .github/workflows/check-advisories.yml]: Very nice idea.

**Review by frasertweedale — APPROVED** [review]

**Comment by frasertweedale** [comment]
@TristanCacqueray thanks for this.  If there are no more changes to be added to this PR, please go ahead and merge it.

**Comment by TristanCacqueray** [comment]
I meant to add the `limit` attribute, and detect unknowns keys (the parser presently ignore them). But that can be done in a follow-up, and this PR already gets us closer to the final schema.

Thank you for the review!

**Pull request merged**